### PR TITLE
docs: Create ADOPTERS.md

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: ADOPTERS.md
Comment (kavirajk) [inline ADOPTERS.md:1]: include Opstrace as well?

**Review by kavirajk — APPROVED** [review]
LGTM!

File: ADOPTERS.md
Comment (RichiH) [reply ADOPTERS.md:1]: Did they agree? The more the merrier! Will merge this one and if you know they agreed, toss them in right after! :)

**Comment by RichiH** [comment]
Ah, I can't merge. @kavirajk do you want to toss in in Opstrace before the merge, then?

**Comment by kavirajk** [comment]
@RichiH it was just a suggestion. feel free to merge :)

**Pull request merged**
